recipes/dmalloc/dmalloc_5.5.2.oe

```diff
@@ -0,0 +1,60 @@
+DESCRIPTION = "Debug Malloc Library"
+HOMEPAGE = "http://dmalloc.com"
+LICENSE = "CC-BY-SA-3.0"
+
+SRC_URI = "\
+  http://dmalloc.com/releases/dmalloc-${PV}.tgz \
+  file://configure-pagesize-HACK.patch \
```

Comment:
ugh, this patch seems like a really bad idea :) could we try figuring out where it is from? same with the other patch